tests/core/io/test_marshalls.h

```diff
@@ -141,7 +141,7 @@ TEST_CASE("[Marshalls] Floating point double precision decoding") {
 }
 
 TEST_CASE("[Marshalls] C string encoding") {
-	char cstring[] = "Godot"; // 5 characters
+	char cstring[] = "Blazium"; // 5 characters
```

Comment:
"Blazium" is a little bit longer than 5 characters.

Reply:
On line 149 there's ca character size check. Checks every character. Didn't touch